lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorType.java

```diff
@@ -30,6 +30,9 @@
    * Indicates an Evaluator that runs on the CLR.
    */
   CLR,
+  /** Indicates an Evaluator that runs on a local thread.
+   */
+   LOCAL,
```

Comment:
This seems like an orthogonal category of `EvaluatorType`. The others are about the runtime environment (JVM vs. CLR), while this one is about the REEF runtime it is on; We have a field for that in `EvaluatorInfo`, IIRC. Maybe this information should move there?